.travis.yml

```diff
@@ -2,17 +2,24 @@ language: ruby
 cache: bundler
 sudo: false
 rvm:
-  - 2.3.6
-  - 2.4.3
-  - 2.5.0
+  - 2.3
+  - 2.4
+  - 2.5
+  - 2.6
```

Comment:
👍

Reply:
This will (probably) ease the work at travis between patch versions of Ruby.